app/controllers/application_controller.rb

```diff
@@ -1,10 +1,39 @@
 require './config/environment'
 
 class ApplicationController < Sinatra::Base
-
   configure do
+    enable :sessions
+    set :session_secret, 'fwitter'
     set :public_folder, 'public'
     set :views, 'app/views'
   end
 
-end
\ No newline at end of file
+  get '/' do
+    erb :index
+  end
+
+  helpers do
+
+    def logged_in?
+      !!session[:user]
+    end
+
+    def current_user
```

Comment:
Generally, the `current_user ` helper method would query the database for the user with the same id as the `session[:id]` and would return that instance. e.g.
``` ruby
def current_user
  User.find_by(id: session[:id])
end
```